src/stream/src/executor/backfill/cdc/cdc_backfill.rs

```diff
@@ -434,20 +443,42 @@ impl<S: StateStore> CdcBackfillExecutor<S> {
                     }
                 }
             }
+        } else {
+            tracing::info!(
+                upstream_table_id,
+                upstream_table_name,
+                "CdcBackfill has been skipped"
+            );
+            state_impl
+                .mutate_state(
+                    current_pk_pos,
+                    last_binlog_offset.clone(),
+                    total_snapshot_row_count,
+                    true,
+                )
+                .await?;
```

Comment:
Notice that `state.is_finished == true` will also enter this `else` block, is it intended? (at least the log message "CdcBackfill has been skipped" looks strange to me)

Reply:
Thanks for pointing out, this is a bug indeed. 😇